.travis.yml

```diff
@@ -28,9 +26,8 @@ before_script:
   - perl Makefile.PL
   - make
 script:
-  - export MQHOST="$LOCAL_MQHOST"
-  - export MQUSERNAME="guest"
-  - export MQPASSWORD="guest"
-  - if [[ "$MQHOST" != "" ]]; then cover -test -report coveralls -ignore_re "\.c|\.h"; fi
-  - if [[ "$MQHOST" != "" ]]; then perl -I blib/lib -I blib/arch xt/100_transaction_memory_leak.t; fi
-  - if [[ "$MQHOST" != "" ]]; then perl -I blib/lib -I blib/arch xt/101_headers_memory_leak.t; fi
+  - if [[ "$(which brew)" == "" ]]; then export MQHOST="localhost"; export MQUSERNAME="guest"; export MQPASSWORD="guest"; fi
+  - cover -make "make TEST_VERBOSE=1" -test test -report coveralls -ignore_re "\.c|\.h"
+  - export MQHOST="localhost"; export MQUSERNAME="guest"; export MQPASSWORD="guest"
+  - if [[ "$(which brew)" == "" ]]; then perl -I blib/lib -I blib/arch xt/100_transaction_memory_leak.t; fi
```

Comment:
I was trying not to tie the exception only to OSX... but I guess Travis only gives us OSX and GNU/Linux.